src/qt/guiutil.cpp

```diff
@@ -49,7 +49,9 @@
 #include <QDateTime>
 #include <QDesktopServices>
 #include <QDesktopWidget>
-#include <QDoubleValidator>
+#include <QRegExp>
+#include <QRegularExpression>
```

Comment:
`#include <QRegExp>` and `#include <QRegularExpression>` seem unnecessary.

Reply:
`QRegExp` is being used in `getSaveFileName` method. `QRegularExpression` in the new edit box validator.